opencv/install_requirements.sh

```diff
@@ -23,4 +23,6 @@ if grep -s -q "MX8MQ" /sys/firmware/devicetree/base/model; then
   fi
 fi
 
-sudo apt install python3-opencv
+sudo pip3 install opencv-contrib-python
```

Comment:
@mtyka FYI this is an unofficial OpenCV package (it's a pre-built version of OpenCV),  but I could also modify to script to do so, it's just a [very involved process](https://www.deciphertechnic.com/install-opencv-python-on-raspberry-pi/)

Reply:
I think this package is only available starting from  Raspbian Buster, the latest version of Raspbian. I just tried it and it work. 
Thus think we should stick with "sudo apt install python3-opencv".
If the unofficial one works with Raspbian Stretch then perhaps just leave a comment here to  that effect.